src/components/Tooltip/TooltipRenderedOnPageBody.js

```diff
@@ -65,7 +63,7 @@ class TooltipRenderedOnPageBody extends React.PureComponent {
         super(props);
         this.state = {
             // The width of tooltip's inner content
-            tooltipContentWidth: 0,
+            tooltipContentWidth: undefined,
```

Comment:
Why switching to `undefined`?

Reply:
because otherwise the tooltip style will have `width: 0`, thus the content has no space to render, and thus we can't measure the content. Thus on the first render the width has to be undefined.

Reply:
Ah I see, Thanks. But I guess this will introduce a visual flicker effect (tooltip changing width)

Reply:
I think we can fix that by applying invisible style while `tooltipContentWidth` is `undefined`